src/config.hpp

```diff
@@ -285,6 +285,15 @@ class Config {
         return result->operator[]("metrics_enabled").as<bool>();
     }
 
+    /**
+     * @brief Check if result was parsed
+     *
+     * @return bool
+     */
+    bool isResultParsed() {
```

Comment:
```suggestion
    bool isParsed() {
```
Result cannot be parsed, result is available once parsing is complete.

Reply:
Done.